src/Symfony/Component/Translation/TranslatorBagInterface.php

```diff
@@ -30,4 +26,11 @@ interface TranslatorBagInterface
      * @throws InvalidArgumentException If the locale contains invalid characters
      */
     public function getCatalogue(string $locale = null): MessageCatalogueInterface;
+
+    /**
+     * Returns all catalogues of the instance.
```

Comment:
I think we can safely remove this short description, it doens't add anything.

Reply:
on contracts it's worth being extra clear I think